angular-daemon/package.json

```diff
@@ -1,39 +1,47 @@
 {
   "name": "angular-daemon",
   "description": "Amorphic daemon mode playground application",
+  "engines": {
+    "node": ">=12.0.0",
+    "npm": ">=6.0.0"
+  },
   "scripts": {
-    "start": "nodemon ./dist/server.js",
-    "debug": "node --inspect ./dist/server.js",
+    "build": "npm run compile && npm run copyStatics",
     "compile": "tsc",
-    "compile:watch": "tsc  --watch",
-    "lint": "tslint --project ./tsconfig.json --config ./tslint.json './apps/**/*.ts'",
+    "compile:watch": "tsc --watch",
     "copyStatics": "cp config.json ./dist/ && cp apps/note/*.json dist/apps/note/ ",
-    "test": "mocha ./dist/test/unit/**/*.js",
-    "coverage": "nyc mocha ./dist/test/unit/**/*.js"
+    "debug": "node --inspect ./dist/server.js",
+    "lint": "eslint ./apps/**/*.ts --quiet",
+    "start": "nodemon ./dist/server.js",
+    "test": "mocha -r ts-node/register ./test/unit/**/*.ts",
+    "test:coverage": "nyc npm run test"
   },
   "private": true,
   "dependencies": {
-    "amorphic": "4.0.0",
+    "@havenlife/persistor": "^4.0.0",
+    "@havenlife/semotus": "^3.2.1",
+    "@havenlife/supertype": "^3.3.1",
```

Comment:
I don't think you should specify these here. Amorphic will pull its own dependencies and as a result may clash with these.

Reply:
removed